### PR TITLE
fix: update makefile indents to not always run which

### PR DESCRIPTION
**Pull request by powersj**
Currently, any make command will run these three which commands because of how they are indented. It is annoying if you don't happen to have one of the items installed. This moves them back to where they actually need to be run.

### Required for all PRs:

- [ ] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [ ] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

resolves #

### CODE REVIEW

**Comment by srebhan** [comment]
@powersj  while you are at it, the suggested [markdown linter](https://www.npmjs.com/package/markdownlint-cli) installs a binary called `markdownlint` and **not** `markdownlint-cli`. So I still get the error even after running `make lint-install`.

**Comment by powersj** [comment]
Pushed an update with that fix as well

**Review by srebhan — APPROVED** [review]
Perfect. Thank you @powersj!

**Review by reimda — APPROVED** [review]

**Comment by srebhan** [comment]
I see the following error with this PR merged that was not there before:
```shell
> make lint-branch
golangci-lint can't be found, please run: make lint-install
ifeq (, /home/sven/go/bin/golangci-lint)
/bin/sh: -c: line 1: syntax error near unexpected token `,'
/bin/sh: -c: line 1: `ifeq (, /home/sven/go/bin/golangci-lint)'
make: *** [Makefile:167: lint-branch] Error 2
```
When unindenting the (shell) code under `lint-branch` the error goes away...

File: Makefile [hunk 1]
```diff
@@ -148,26 +148,26 @@ lint-install:
...
-ifeq (, $(shell which markdownlint-cli))
-	$(info markdownlint-cli can't be found, please run: make lint-install)
-	exit 1
-endif
+	ifeq (, $(shell which markdownlint))
+		$(info markdownlint can't be found, please run: make lint-install)
+		exit 1
+	endif
 
 	markdownlint-cli
```
Comment (Hipska) [inline Makefile:163]: Isn't that contradictory? It checks for `markdownlint` but then runs `markdownlint-cli`.

Comment (powersj) [reply Makefile:163]: Ah yes, I need to change it in one more place. I was focused on the odd indented sections. Good catch

**Pull request merged**